src/test/java/io/xstefank/wildfly/bot/PRConfigFileChangeTest.java

```diff
@@ -76,8 +70,6 @@ void testUpdateIncorrectlyIndentedFile() throws IOException {
                     GHCommitState.ERROR, "", "Unable to parse the configuration file. " +
                         "Make sure it can be loaded to model at https://github.com/xstefank/wildfly-github-bot/blob/main/CONFIGURATION.yml",
                     "Configuration File");
-
-                verifyNoMoreInteractions(mocks.pullRequest(gitHubJson.id()));
```

Comment:
why?

Reply:
I was using the other PR's commit and removing the unnecessary changes. This one slipped between the formatting changes